zScanner/Coordinators/NewDocumentCoordinator.swift

```diff
@@ -91,6 +91,8 @@ class NewDocumentCoordinator: Coordinator {
     }
     
     private func finish() {
+        #warning("TODO Should we setting a timezone?")
+        newDocument.date = Date()
```

Comment:
I'm not sure what time should we put here. We might be using the time of the creation of the image from the gallery. Or did Martin already specified this?

Reply:
Martin created a ticket with a question if we are able to get metadata of photos from the gallery. I found a solution on StackOverflow but I couldn't try it because I can't run the app on my mobile.

Reply:
We don't know what is the desired solution yet. Remove these lines for now and let's just move on with removing the date time picker from the screen.

Reply:
Martin wrote that the time of validity is a time when I press the send button.